### PR TITLE
feat(ordering): execute commands using correct priority order

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: tests/unit/070_job_processes.spec.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { prepareCommands } from '../../src/api/process';
+import { CommandType, Command, CommanyTypePriority } from '../../src/api/config';
+/* import { JobProcess } from '../../src/api/process-manager'; */
```
Comment (jkuri) [inline tests/unit/070_job_processes.spec.ts]: please remove comments.

File: tests/unit/070_job_processes.spec.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { prepareCommands } from '../../src/api/process';
+import { CommandType, Command, CommanyTypePriority } from '../../src/api/config';
```
Comment (jkuri) [inline tests/unit/070_job_processes.spec.ts]: `CommanyTypePriority` I believe you have a typo here.

File: tests/unit/070_job_processes.spec.ts (hunk 1 above)
Comment (Izak88) [reply tests/unit/070_job_processes.spec.ts]: done

File: tests/unit/070_job_processes.spec.ts (hunk 2 above)
Comment (Izak88) [reply tests/unit/070_job_processes.spec.ts]: done

**Pull request merged**